testutil/multi_tenancy.go

```diff
@@ -411,3 +411,15 @@ export(input: {destination: $dst, format: $f, accessKey: $acc, secretKey: $sec})
 	require.NoError(t, err)
 	return resp
 }
+
+func AddNumberOfTriples(t *testing.T, dg *dgo.Dgraph, start, end int) (*api.Response, error) {
```

Comment:
this function should just be defined in the test code itself. This doesn't seem like a common function

Reply:
@mangalaman93 We can use this function in future test where we need to add number of triplets.